components/brave_wallet/browser/zcash/zcash_shield_sync_service.cc

```diff
@@ -34,8 +36,14 @@ size_t GetCode(ZCashShieldSyncService::ErrorCode error) {
       return 5;
     case ZCashShieldSyncService::ErrorCode::kFailedToRetrieveAccount:
       return 6;
-    case ZCashShieldSyncService::ErrorCode::kScannerError:
+    case ZCashShieldSyncService::ErrorCode::kFailedToVerifyChainState:
       return 7;
+    case ZCashShieldSyncService::ErrorCode::kFailedToUpdateSubtreeRoots:
+      return 8;
+    case ZCashShieldSyncService::ErrorCode::kDatabaseError:
+      return 9;
+    case ZCashShieldSyncService::ErrorCode::kScannerError:
+      return 10;
   }
```

Comment:
would that be enough to just `return static_cast<size_t>(error);` ? 
And why it returns `size_t`?

Reply:
It was enum class, reworked using simple enum